ftp/1.5/modules/ROOT/pages/ftp-connection.adoc

```diff
@@ -0,0 +1,36 @@
+= Connect to an FTP Server Example - Mule 4
+
+To configure the global element to connect to an FTP Server, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to *Connector configuration* to access the global element configuration fields.
+. On the *General* tab, specify the connection information for the connector.
+. Set *Working Directory* to `${workingDir}`. +
+This is root of every relative path used with this connector. If it's not provided, the value defaults to the remote server.
```

Comment:
"the root"

Reply:
Just an editorial comment - to use fewer words, you can remove the last sentence and say "The value defaults to the remote server."